components/greaselion/browser/greaselion_service_impl.cc

```diff
@@ -139,6 +143,17 @@ scoped_refptr<Extension> ConvertGreaselionRuleToExtensionOnTaskRunner(
     return nullptr;
   }
 
+  // Copy the messages directory to our extension directory.
+  if (!rule->messages().empty()) {
+    if (!base::CopyDirectory(
+            rule->messages(),
+            temp_dir.GetPath().Append(rule->messages().BaseName()), true)) {
```

Comment:
force this directory name to be `_locales` since anything else won't work?

Reply:
Good point, fixed.